.github/workflows/inductor.yml

```diff
@@ -188,6 +188,13 @@ jobs:
           { config: "cpu_aot_inductor_torchbench_freezing", shard: 1, num_shards: 2, runner: "linux.12xlarge" },
           { config: "cpu_aot_inductor_torchbench_freezing", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
           { config: "inductor_torchbench_cpu_smoketest_perf", shard: 1, num_shards: 1, runner: "linux.24xl.spr-metal" },
+          { config: "inductor", shard: 1, num_shards: 2, runner: "linux.10xlarge.avx2" },
```

Comment:
Can you add avx2 in the config names to make them more distinguishable?

Reply:
Thanks for the review. Added avx2 in the config names.